mux.go

```diff
@@ -253,3 +252,24 @@ func camelToHuman(s string) string {
 	}
 	return result.String()
 }
+
+// DefaultDescription returns a default .md description for a controller
+func DefaultDescription[T any](handler string, middlewares []T) string {
+	description := "#### Controller: \n\n`" +
+		handler + "`"
+
+	if len(middlewares) > 0 {
+		description += "\n\n#### Middlewares:\n"
+
+		for i, fn := range middlewares {
+			description += "\n- `" + FuncName(fn) + "`"
+
+			if i == 4 {
+				description += "\n- more middleware..."
```

Comment:
```suggestion
				description += "\n- more middleware…"
```

Unicode is your friend